package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "records-server",
+  "version": "1.0.0",
+  "description": "An open source project to practice client-server intergration",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We need a script for to run `eslint`, in the client we called `lint`, maybe we could call it the same here, what do you think?

Reply:
done, implemented

Reply:
Thanks for that.